src/hooks/index.js

```diff
@@ -123,6 +124,15 @@ function activate(agent) {
 
   logger = agent.logger;
 
+  // Plugin stuff
+  var api = new Plugin(agent);
+  Object.keys(agent.plugins).forEach(function(pluginName) {
+    if (toInstrument[pluginName]) {
```

Comment:
Added new file `trace-plugin-loader.js`, and this file was reverted to original.